pylintrc

```diff
@@ -52,7 +52,8 @@ confidence=
 # no Warning level messages displayed, use"--disable=all --enable=classes
 # --disable=W"
 
-disable=invalid-name,protected-access,fixme,too-many-branches,
+disable=implicit-str-concatenation-in-literal-iterable,
```

Comment:
I assume it's disabled here due to message definitions in all checkers, where implicitly joined strings for description are used extensively...

Reply:
Yes :)